src/components/MoneyReportHeaderStatusBar.tsx

```diff
@@ -14,7 +14,7 @@ import RenderHTML from './RenderHTML';
 
 type MoneyReportHeaderStatusBarProps = {
     /** The next step for the report */
-    nextStep: ReportNextStep;
+    nextStep: ReportNextStep | undefined;
```

Comment:
When will next step be undefined? / Why do we need this now? Shouldn't the next step always be determined?